### PR TITLE
Add steps for creating AAD tenant for dev environments

### LINKED ISSUE #1185: [Task] Update docs with steps for using a separate AAD tenant
**Description**
Update the installation documentation to include steps for creating a separate AAD tenant for dev environments

### PR DESCRIPTION
**Pull request by stuartleeks**
Adding a note to docs on how to create a separate AAD tenant, e.g. for working in a dev environment where you don't have permissions to create the AAD app

Closes #1185

### CODE REVIEW

File: docs/tre-admins/setup-instructions/pre-deployment-steps.md [hunk 1]
```diff
@@ -56,14 +56,14 @@ Next, you will set the configuration variables for the specific Azure TRE instan
...
 
   ```bash
   ./scripts/aad-app-reg.sh -n TRE -r https://mytre.westeurope.cloudapp.azure.com/api/docs/oauth2-redirect -a
   ```
 
   !!! note
-      The full functionality of the script requires directory admin privileges. You may need to contact your friendly AAD admin to complete this step. The app registrations can be created manually in Azure Portal too. For more information, see [Authentication and authorization](../auth.md).
+      The full functionality of the script requires directory admin privileges. You may need to contact your friendly Azure Active Directory admin to complete this step. The app registrations can be created manually in Azure Portal too. For more information, see [Authentication and authorization](../auth.md).<br/>If you don't have permissions and just want to create a development environment or explore TRE then you can create a separate Azure Active Directory tenant for this stage by [following the steps here](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant) (note that you won't be able to use certain Active Directory integrated services in this configuration, so it is not recommended for production).
```
Comment (marrobi) [inline docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: Do we need to cover connecting to this tenant to create the app registrations? i.e. `--allow-no-subscriptions`

Comment (martinpeck) [inline docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: suggest remove <br/> HTML and simply add two newlines to get the same result

Comment (martinpeck) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: GitHub removed my `<br/>` and replaced it with an actual break! :)

Comment (martinpeck) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: Might be better to change `(note that you....` into a note using `>`

e.g. ...

> Note: You won't be able to use certain...

Comment (stuartleeks) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: I thought about splitting it but I wasn't sure how that would play out with the `!!!note` syntax that is above?

Comment (stuartleeks) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: you mean on the `az login` step?

Comment (marrobi) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md:66]: Yes, as that has been catching people out, then the need to log back into to the target subscription once done the tenant specific stuff.

Comment (stuartleeks) [reply docs/tre-admins/setup-instructions/pre-deployment-steps.md]: I've split this out into a separate section now as there was too much to fit in the note!

**Review by marrobi — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**